pkgs/top-level/aliases.nix

```diff
@@ -620,9 +446,7 @@ mapAliases {
   gnome-hexgl = throw "'gnome-hexgl' has been removed due to lack of maintenance upstream"; # Added 2024-09-14
   gnome-passwordsafe = gnome-secrets; # added 2022-01-30
   gnome_mplayer = throw "'gnome_mplayer' has been removed due to lack of maintenance upstream. Consider using 'celluloid' instead"; # Added 2024-09-14
-  gnome-mpv = throw "'gnome-mpv' has been renamed to/replaced by 'celluloid'"; # Converted to throw 2023-09-10
   gnome-resources = resources; # added 2023-12-10
-  gnome_user_docs = throw "'gnome_user_docs' has been renamed to/replaced by 'gnome-user-docs'"; # Converted to throw 2023-09-10
 
   gnuradio-with-packages = gnuradio3_7.override {
```

Comment:
This doesn't eval anymore.

```
       error: undefined variable 'gnuradio3_7'

       at /home/runner/work/nixpkgs/nixpkgs/pkgs/top-level/aliases.nix:451:28:

          450|
          451|   gnuradio-with-packages = gnuradio3_7.override {
             |                            ^
          452|     extraPackages = lib.attrVals [
```

`gnuradio3_7` alias was removed further down.

Reply:
Yeah, sorry, jumped the gun because I trusted our tooling and wasn’t sure how to exhaustively test `aliases.nix` eval locally (and forgot that `nixpkgs-vet` catches eval errors fast these days). My bad. It’s being fixed in https://github.com/NixOS/nixpkgs/pull/350291.

Reply:
We should probably figure out a way to stop aliases pointing to other aliases since it seems like a footgun.